database/schema.sql

```diff
@@ -184,7 +184,8 @@ CREATE TABLE `configuration` (
 LOCK TABLES `configuration` WRITE;
 INSERT INTO `configuration` (field, value, description) VALUES("game", "0", "(Boolean) Game is ongoing");
 INSERT INTO `configuration` (field, value, description) VALUES("next_game", "0", "(Date) Next game to happen");
-INSERT INTO `configuration` (field, value, description) VALUES("game_duration", "10800", "(Integer) Duration of game in seconds");
+INSERT INTO `configuration` (field, value, description) VALUES("game_duration_value", "3", "(Integer) Value of the duration of the game");
```

Comment:
Doesn't this code belong to a different commit that was just landed this week? The one for the custom duration: https://github.com/facebook/fbctf/commit/bd8db7ab8b560331898a19fe5bae463cabea72b0